pyscf/nao/chi0_matvec.py

```diff
@@ -82,7 +82,7 @@ def __init__(self, **kw):
     self.moms0,self.moms1 = pb.comp_moments(dtype=self.dtype)
     self.nprod = self.moms0.size
 
-    if self.verbosity>0 : print(__name__, ' nprod ', self.nprod, ' cc_da.shape ', self.cc_da.shape)
+    if self.verbosity>0 : print(__name__,'\t====> Number of products = {} and their shape = {}'.format(self.nprod, self.cc_da.shape))
```

Comment:
Here, you may rename it to 'Number of dominant products {} and Number of atom-centered products {}'.format(shape[0], shape[1])